api/models.py

```diff
@@ -62,3 +62,15 @@ class Task (models.Model):
 	deadline = models.DateTimeField(blank=True, null=True)
 	date_created = models.DateTimeField(auto_now_add=True)
 	date_updated = models.DateTimeField(auto_now=True)
+
+class Event(models.Model):
+
+	title = models.CharField(max_length=255, blank=False, null=False)
+	start_date = models.DateTimeField(blank=False, null=False)
+	end_date = models.DateTimeField()
+	location = models.CharField(max_length=1000, blank=False)
+	description = models.CharField(max_length=1000)
+	organizer = models.CharField(max_length=255)
+	date_created = models.DateTimeField(auto_now_add=True)
+	date_updated = models.DateTimeField(auto_now_add=True)
```

Comment:
Not, just one. Read again in the docs, what the difference is: https://docs.djangoproject.com/en/1.7/ref/models/fields/#datefield